packages/apps/tools/src/constants/kadena.ts

```diff
@@ -1,41 +1,46 @@
 import { env } from '@/utils/env';
+import type { ChainwebNetworkId } from '@kadena/chainweb-node-client';
+
+export type NetworkNames = Exclude<ChainwebNetworkId, 'development'>;
+export type NetworkName = NetworkNames | string;
```

Comment:
Why should this exclude 'development'? It is a valid networkType.

Reply:
WRT the naming of the type, I'd opt for `NetworkID` as type, since it's also that in the `chainweb-node-client` library. 
When not excluding `development` you can even use `ChainwebNetworkId` everywhere as **Typescript type**.

Reply:
Whenever there's a change in the networks, it would propagate straight from `chainweb-node-client` into this app, and allow us to update accordingly